<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchAppointmentDialog</class>
 <widget class="QDialog" name="SearchAppointmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Записи - Поиск</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayoutAppointmentForm">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButtonAppointmentDoctorPhone">
       <property name="text">
        <string>Номер врача</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButtonAppointmentPatientPhone">
       <property name="text">
        <string>Номер пациента</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayoutAppointmentDatetime">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutAppointmentDatetimeDate">
         <item>
          <widget class="QSpinBox" name="spinBoxAppointmentDatetimeDay">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>31</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxAppointmentDatetimeMonth">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxAppointmentDatetimeYear">
           <property name="minimum">
            <number>2012</number>
           </property>
           <property name="maximum">
            <number>2023</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutAppointmentDatetimeTime">
         <item>
          <widget class="QComboBox" name="comboBoxAppointmentDatetimeHour"/>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxAppointmentDatetimeMinute"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditAppointmentDoctorPhone"/>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="radioButtonAppointmentCost">
       <property name="text">
        <string>Стоимость</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditAppointmentPatientPhone"/>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButtonAppointmentDatetime">
       <property name="text">
        <string>Дата и Время приёма</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBoxAppointmentCost">
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutActions">
     <item>
      <widget class="QPushButton" name="pushButtonSearch">
       <property name="text">
        <string>Искать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
